lib/src/parse.cpp

```diff
@@ -999,10 +1009,32 @@ const char* object_set::parse_objects(const char* cur) noexcept (false) {
                 patch_missing_value( attr );
             }
 
+            object_clear = object_clear && is_log_clear(attr.log);
+
             current.set(attr);
         }
 
+        if (not object_clear) {
+            const auto msg =
+                "Some object attributes violate specification, which might "
```

Comment:
1. I agree that my suggestion was not the best. But I'm still unsure if the current message is clear enough. Maybe we should simply say `This object is a part of a object-set that violated the specification`. Then as this is just info, we don't explicitly need to say how trustworthy it is?
2. We also set a similar message for the objects-set. Currently we log the set-level when querying E.g. `f.object`, `f.find`. Then when someone tries to _use_ the object, we log the object-level msg. Consider the normal use case:
```
obj = f.object(...)
_ obj.attribute
```
that will issue two almost identical messages, but even to me it's not completely obvious that they refer to the _exact_ same issue.
```INFO     root:errorhandler.py:141 
Severity:     minor.
Where:        object set of type 'VERY_MUCH_TESTY_SET' named 'SETNAME'.
Problem:      Some objects in this object set violate specification, which might mean that correctness of other objects is also affected.
INFO     root:errorhandler.py:141 
Severity:     minor.
Where:        T.VERY_MUCH_TESTY_SET-I.OBJECT-O.1-C.1.
Problem:      Some object attributes violate specification, which might mean that correctness of other attributes is also affected. 
```
So my question is, do we need to set this information on the set-level? Does it bring any value or is it just extra noise and confusion? I'm leaning towards the last. And I don't see a big problem with the user being delivered a "broken" object without a warning, as long as the warning comes when they try to use the object. After all, that's how we deal with all other objects.

(This time I'm not arguing _where_ the set-level messages should be issued from, just it _this specific_ message is needed)

Reply:
Im still not to fuzzed about this error msg, but other than that I'm happy

> some object attributes violate specification, which might mean that correctness of other attributes is also affected

It's to wage for me, and that causes confusion and a bit of frustration when reading it. (why don't I understand this, it seams like something I such understand as there is no unfamiliar concepts)
> some object attributes 

what does that means, from _this_ object, from another object somewhere else, can a parameter attribute fuck up a channel attribute like this? 

> correctness of other attributes

Which attributes are we talking about now, the once on _this_ object, or is it more general?

In this case we have the option to be either too specific (e.g. users generally don't know what a set is) or too wage which cases equally, if not more confusion. 

I would say:
"One or more attributes on this object violated the spec, this can potentially corrupt the entire object" 
Or something to that effect